src/main/java/seedu/address/model/dayData/CustomQueue.java

```diff
@@ -243,30 +243,34 @@ public int hashCode() {
     }
 
     /** Returns true if {@code dayDatas} table constraints are enforced */
-    private boolean tableConstraintsAreEnforced(List<DayData> dayDatas) {
-        int count = 0;
+    public static boolean tableConstraintsAreEnforced(List<DayData> dayDatas) {
+        boolean res = true;
```

Comment:
also, could replace all the res = false with exceptions? then you can state explicitly what the error was

Reply:
I followed the format for TaskList, apparently this method is used to determine whether an exception should be thrown or not. I will check and get back to you again!